Sources/LiveViewNative/Modifiers/Navigation Modifiers/NavigationBarTitleDisplayModeModifier.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  NavigationBarTitleDisplay.swift
+//  LiveViewNative
+//
+//  Created by Dylan.Ginsburg on 4/24/23.
+//
+
+import SwiftUI
+
+/// Configures the title display mode for this view.
+///
+/// ```html
+/// <VStack
+///     modifiers={
+///         @native
+///             |> navigation_title:(title: "Live View Native")
+///             |> navigation_bar_title_display_mode(display_mode: :large)
```

Comment:
I might be mistaken, but it since like `display_mode: large` in the docs is not one of the  possible values :sweat_smile:. 
apologies for the noise

Reply:
no apologies necessary, I love that people are paying attention to this :)

Reply:
According to the docs, it does appear supported: https://developer.apple.com/documentation/swiftui/navigationbaritem/titledisplaymode#topics

Reply:
you are right, I totally missed it.